core/extension/native_extension_manager.cpp

```diff
@@ -69,9 +69,9 @@ NativeExtensionManager::LoadStatus NativeExtensionManager::unload_extension(cons
 		if (minimum_level < MIN(level, NativeExtension::INITIALIZATION_LEVEL_SCENE)) {
 			return LOAD_STATUS_NEEDS_RESTART;
 		}
-		//initialize up to current level
+		// deinitialize down to current level
```

Comment:
While at it:
```suggestion
		// Deinitialize down to current level.
```
And you can do the same for the init comment.

Reply:
Fixed.